src/components/Share/Share.tsx

```diff
@@ -1,25 +1,32 @@
-import { FC } from 'react'
+import { FC, useMemo } from 'react'
 import Link from 'next/link'
+import { Address } from 'wagmi'
 
 import { TwitterIcon } from '@/components/icons/Twitter'
 import { useMintDialogContext } from '@/components/MintDialog/Context/useMintDialogContext'
+import { website } from '@/config/website'
 import { WarpCast } from '@/components/icons/Warpcast'
 import { Lens } from '@/components/icons/Lens'
+import { useEns } from '@/utils/useEns'
+import { schedule } from "@/config/schedule";
 
 const twitterURL: string = 'https://twitter.com/intent/tweet'
 const warpCastURL: string = 'https://warpcast.com'
 const lensURL: string = 'https://lenster.xyz'
 
 type ShareComponentProps = {}
 export const Share: FC<ShareComponentProps> = () => {
-  const { dropName } = useMintDialogContext()
-  const {
-    location: { href },
-  } = window
+  const { dropName, partnerName, creatorAddress } = useMintDialogContext()
+  const { name } = useEns(creatorAddress as Address)
```

Comment:
If creator address doesn't resolve in useENS, this will return `''`. Can you modify the messaging below to be `name || creatorAddress`

Reply:
Done the change

Reply:
Sorry @arpit2438735 - you used nullish coalescing, which won't resolve the issue

if name were `undefined` or `null` your solution would work, but where the name will return as the empty string `''`, your nullish coalescing will behave differently than the suggested `name || creatorAddress`

Reply:
yeah you are right, my bad I thought name will be null